ubuntu20.04/val-verde-platform-sdk-libcxx-cross

```diff
@@ -58,7 +58,8 @@ ${BUILD_PACKAGE_PREFIX}/bin/${PACKAGE_BASE_NAME}-platform-sdk-cmake \
 source ${BUILD_PACKAGE_PREFIX}/bin/${PACKAGE_BASE_NAME}-platform-sdk-ninja-build
 
 # Build the package
-source ${BUILD_PACKAGE_PREFIX}/bin/${PACKAGE_BASE_NAME}-platform-sdk-package-build
+DEPENDENCIES=${PACKAGE_BASE_NAME}-libcxxabi-${HOST_OS}${HOST_OS_API_LEVEL}-${HOST_ARCH}
```

Comment:
This variable is an argument to the script so we want to escape it so that its contents are leaked to the rest of this script.
```
DEPENDENCIES=${PACKAGE_BASE_NAME}-libcxxabi-${HOST_OS}${HOST_OS_API_LEVEL}-${HOST_ARCH} \
source ${BUILD_PACKAGE_PREFIX}/bin/${PACKAGE_BASE_NAME}-platform-sdk-package-build "${DEPENDENCIES}"
```
Notice the \ at the end of the DEPENDENCIES.

Reply:
removed this file to address deb generation as a separate PR